wormhole-connect/src/views/v2/Bridge/Routes/SingleRoute.tsx

```diff
@@ -242,7 +242,7 @@ const SingleRoute = (props: Props) => {
   }
 
   return (
-    <div key={routeName} className={classes.container}>
+    <div key={name} className={classes.container}>
```

Comment:
We need to use the name prop here as route can be same for automatic and manual of the same route.